kustomize/base/all-in-one/resources/deploy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: event-bridge
+spec:
+  selector:
+    matchLabels:
+      app: event-bridge
+  replicas: 1
+  template:
+    spec:
+      containers:
+        - image: event-bridge-all-in-one
+          imagePullPolicy: Always
```

Comment:
Can be `IfNotPresent`. We're using a unique image tag for each build of our container.